src/js/airtable-list-builder.js

```diff
@@ -358,4 +502,19 @@
     $('#airtable-list').height(newTop);
   }
 
+  // debounce so filtering doesn't happen every millisecond
+  function debounce( fn, threshold ) {
```

Comment:
Drupal has a debounce https://drupaljs.net/10.0.x/debounce.js.html

Reply:
Ehh Good point. This is probably a future improvement.